ddtrace/tracer.py

```diff
@@ -315,7 +315,8 @@ def configure(
         elif writer is None and isinstance(self.writer, LogWriter):
             # No need to do anything for the LogWriter.
             pass
-        self.writer.dogstatsd = get_dogstatsd_client(self._dogstatsd_url)
+        if isinstance(self.writer, AgentWriter):
```

Comment:
Also seems like an unrelated change

Reply:
Another mypy caught bug.
Not all writer have `dogstatsd` as an attribute.